Start mentioning trailers. See #884. (#1157)
httpwg/http-extensions: draft-ietf-httpbis-digest-headers.md

knowing whether the recipient supports the digest-algorithm, or even knowing
that the recipient will ignore it.

> Digest can be used in trailers. When using incremental digest-algorithms
> this allows the sender and the receiver to dynamically compute the digest value
> while streaming the content.

Reviewer: ```suggestion
Digest can be sent in a trailer section. When using incremental digest-algorithms
this allows the sender and the receiver to dynamically compute the digest value
while streaming the content.
```

A `Digest` field using NOT RECOMMENDED digest-algorithms SHOULD NOT be used in
signatures.

## Usage in trailers

When used in trailers, the receiver gets the digest value after the payload body
and may thus be tempted to process the data before validating the digest value.

> Instead, data should be always processed after checking that Digest has the expected value.

Reviewer: ```suggestion
Instead, data should only be processed after validating the Digest.
```

do you want to make this should a normative requirement?

A `Digest` field using NOT RECOMMENDED digest-algorithms SHOULD NOT be used in
signatures.

## Usage in trailers

When used in trailers, the receiver gets the digest value after the payload body
and may thus be tempted to process the data before validating the digest value.
Instead, data should only be processed after validating the Digest.

> If received in trailers, Digest MUST NOT be discarded;

Author: @LPardue sorry Lucas, could you please check this part I stumbled upon re-reading http-messaging-latest?

instead it MAY be merged in the header section (See Section 7.1.2 of {{!MESSAGING=I-D.ietf-httpbis

> -messaging}}).

Author: @LPardue sorry Lucas, could you please check this part I stumbled upon re-reading http-messaging-latest?

Reviewer: lgtm
